src/main/deliferoo/controller/Controller.java

```diff
@@ -1,5 +1,115 @@
 package controller;
 
+import java.io.File;
+import java.util.List;
+
+import model.BestPath;
+import model.Delivery;
+import model.FullMap;
+import view.Window;
+
+/**
+ * Class that implements the MVC application controller
+ * 
+ * @author sadsitha
+ *
+ */
 public class Controller {
 
+    private Window window;
+    private State currentState;
+    private CommandList commandList;
+    private FullMap currentMap;
+    private List <BestPath> round;
+    private List<Delivery> deliveries;
+    protected final InitState initState = new InitState();
+    protected final AddDeliveryState addDeliveryState = new AddDeliveryState();
+    protected final DeliverySelectedState deliverySelectedState = new DeliverySelectedState();
+    protected final MapLoadedState mapLoadedState = new MapLoadedState();
+    protected final DeliveriesLoadedState deliveriesLoadedState = new DeliveriesLoadedState();
+    protected final ModifyDeliveryState modifyDeliveryState = new ModifyDeliveryState();
```

Comment:
Change variable names to all caps, for example : INIT_STATE or STATE_INIT since they are final